### DISCUSSION TITLE
Interested in contributing

### DISCUSSION THREAD

**Discussion by rkleee** [inkstitch/inkstitch#2015] [General]
Hi,
I would be interested in contributing to Inkstitch. I've got an embroidery machine, some experience in Python (but my current main language is Java) and using inkscape. 

Greetings
rkleee

**Comment by kaalleen** [comment]
That's nice, welcome to the project and thank you for your offer.

If you see something in Ink/Stitch that needs improvement or if you can think of any new feature you can just start with that.
Otherwise you can scroll through our list of feature requests and bug reports.

On our website we stored some information about manual installs, code style and debugging: https://inkstitch.org/developers/introduction/

If you need anything else to get started please contact us and we try to help as good as we can.

_Edit:_ Forgot the link to the feature requests and bug reports...

Bugs: https://github.com/inkstitch/inkstitch/projects/2
Feature requests: https://github.com/inkstitch/inkstitch/projects/1
